code/modules/events/stray_cargo.dm

```diff
@@ -96,6 +141,22 @@
 	max_occurrences = 1
 	earliest_start = 30 MINUTES
 	description = "A pod containing syndicate gear lands on the station."
+	admin_setup = /datum/event_admin_setup/stray_cargo/syndicate
+
+/datum/event_admin_setup/stray_cargo/syndicate
+
+/datum/event_admin_setup/stray_cargo/syndicate/override_contents()
+	var/datum/supply_pack/misc/syndicate/custom_value/syndicate_pack = new
+	var/pack_telecrystals = tgui_input_number(usr, "Please input crate's value in telecrystals.", "Set Telecrystals.", 30)
+	if(isnull(pack_telecrystals))
+		pack_telecrystals = 30
+	var/list/possible_uplinks = list("Traitor" = UPLINK_TRAITORS, "Nuke Op" = UPLINK_NUKE_OPS, "Clown Op" = UPLINK_CLOWN_OPS)
+	var/uplink_type = tgui_input_list(usr, "Choose uplink to draw items from.", "Choose uplink type.", possible_uplinks)
+	var/selection = possible_uplinks[uplink_type]
+	if(isnull(selection))
+		selection = UPLINK_TRAITORS
```

Comment:
```suggestion
	var/selection
	if(!isnull(uplink_type))
		selection = possible_uplinks[uplink_type]
	else
		selection = UPLINK_TRAITORS
```

Little bit longer, but I think it's better in how it handles for accessing a potential null in a list (no need to do that list operation needlessly if we're going to account for tgui_input_list returning a null)